addons/event/views/event_views.xml

```diff
@@ -552,30 +557,23 @@
-                                <div class="oe_kanban_details">
+                            <div class="oe_kanban_card oe_kanban_global_click">
+                                <div class="float-right">
+                                    <a class="btn btn-sm btn-success" string="Confirm Registration" name="confirm_registration" type="object" states="draft" role="button">
+                                        <i class="fa fa-check" role="img" aria-label="Confirm button" title="Confirm Registration"/>
+                                    </a>
+                                    <a class="btn btn-sm btn-success" string="Attended the Event" name="action_reg_close" type="object" states="open" role="button">
+                                        <i class="fa fa-level-down" role="img" aria-label="Attended button" title="Attended the Event"/>
+                                    </a>
+                                </div>
+                                <div class="oe_kanban_content">
                                     <div class="o_kanban_record_top">
-                                        <div class="o_kanban_record_headings">
-                                            <strong class="o_kanban_record_title">
-                                                <field name="name"/>
-                                            </strong>
-                                        </div>
-                                        <div class="float-right">
-                                            <a class="btn btn-sm btn-success" string="Confirm Registration" name="confirm_registration" type="object" states="draft" role="button">
-                                                <i class="fa fa-check" role="img" aria-label="Confirm button" title="Confirm Registration"/>
-                                            </a>
-                                            <a class="btn btn-sm btn-success" string="Attended the Event" name="button_reg_close" type="object" states="open" role="button">
-                                                <i class="fa fa-level-down" role="img" aria-label="Attended button" title="Attended the Event"/>
-                                            </a>
-                                            <a class="btn btn-sm btn-danger" string="Cancel Registration" name="button_reg_cancel" type="object" states="draft,open" role="button">
-                                                <i class="fa fa-times" role="img" aria-label="Cancel button" title="Cancel Registration"/>
-                                            </a>
-                                        </div>
+                                        <b class="o_kanban_record_title"><field name="display_name"/></b>
+                                    </div>
+                                    <div class="o_kanban_record_body" style="height: 40px;">
+                                        <field name="event_name"/>
```

Comment:
Why not directly using event_id ? Kanban view will automatically display its name.

Reply:
@tde-banana-odoo 
Because it will use the display name, and so we will have the date at the end of the name of the event (and it takes a lot of space in the kanban card)

Reply:
Then maybe add a context key in name_get of event, either to activate or deactivate the complex name_get . That way we have a generic way of having access to an event "short" name.

Reply:
@tde-banana-odoo 
Don't you think that adding a context key add unnecessary complexity ? (key in the name_get, in the view itself,... instead of just a related field)

Reply:
Using event_id makes a real m2o that you can click.

Indeed context keys are not the best solution.

Best solution: remove that horrible override coming from fa3301735e9bf6b35a0c4a62e149e8a6781858c3 . @qdp-odoo do you remember what you did on July 9 2012 :D ?

Reply:
was it a Thursday?

Reply:
It was a Monday. Did you eat pasta ?

Reply:
I remember clearly: the weather was fine so we opted for a fresh salad instead of the traditional Monday's pasta

:green_salad: 
:spaghetti: